src/main/java/Quill.java

```diff
@@ -0,0 +1,127 @@
+import quill.exception.EmptyDateException;
+import quill.exception.QuillException;
+import quill.task.Deadline;
+import quill.task.Event;
+import quill.task.Task;
+import quill.task.Todo;
+
+import java.util.Scanner;
+public class Quill {
+    public static final String horizontalLine = "____________________________________________________________";
```

Comment:
You could consider renaming (capitalizing) the string literal to follow this code quality [guideline](https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers)